constants.py

```diff
@@ -17,11 +17,16 @@
 # pylint: disable=invalid-name
 
 """Loads constants for backend use."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
```

Comment:
Here and elsewhere: if all the py2/py3 deviations are contained within python_utils, why do we still need these imports in every file? Can they be removed? If not, why not?

Reply:
These imports are required in every file to report incompatible changes(with Python2/3) in that file.

Reply:
For example, the print import will enforce the usage of print function in python 3.
The division import will make the divisions behave as python 3's division

Reply:
Done